config/locales/models.fr.yml

```
@@ -756,6 +757,7 @@ fr:
     filter_value: Valeur du filtre
     find_my_facility: Votre numéro SIRET
     flags: Options
+    forme_exercice: Forme d'activité
```

```suggestion
    forme_exercice: Forme d’activité
```
c’est de la pinaillade